pkg/ocm/resources.go

```diff
@@ -0,0 +1,43 @@
+package ocm
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/openshift/assisted-service/pkg/commonutils"
+
+	"github.com/openshift/assisted-service/internal/common"
+	"github.com/pkg/errors"
+)
+
+//go:generate mockgen -source=resources.go -package=ocm -destination=mock_resources.go
+type OCMResources interface {
```

Comment:
maybe we should call it OCMAccountsMgmt, just to correlate with:
https://github.com/openshift-online/ocm-sdk-go/tree/master/accountsmgmt

Reply:
I can, just saw that you names `OCMAuthorization` for AccountsMgmts so I thought to be more specific on the part of AMS that we are using.

I don't mind to change if you think it is better tough.

Reply:
The authorization is under a different endpoint (/api/authorizations/), but pull-secret authentication is actually also within accounts_mgmt (/api/accounts_mgmt/v1/token_authorization). So maybe we can just call it OCMSubscriptions?
Not crucial of course.. can tackle it later on.

Reply:
Done